lib/babelish/android2csv.rb

```diff
@@ -8,16 +8,20 @@ def initialize(args = {:filenames => []})
 
     def load_strings(strings_filename)
       strings = {}
-      raise Errno::ENOENT unless File.exist?(strings_filename)
+      # raise Errno::ENOENT unless File.exist?(strings_filename)
+      xml_file = File.open(strings_filename)
 
-      xmlfile = XmlSimple.xml_in(strings_filename)
-      xmlfile['string'].each do |element|
-        if !element.nil? && !element['name'].nil?
-          content = element['content'].nil? ? '' : element['content']
-          strings.merge!({element['name'] => content})
+      parser = Nokogiri::XML(xml_file) do |config|
+        config.strict.noent
+      end
+      parser.xpath('//string').each do |node|
+        if !node.nil? && !node['name'].nil?
+          strings.merge!({node['name'] => node.inner_html})
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.